include/raylib-nuklear.h

```diff
@@ -576,28 +582,46 @@ DrawNuklear(struct nk_context * ctx)
 NK_API void nk_raylib_input_keyboard(struct nk_context * ctx)
 {
     bool control = IsKeyDown(KEY_LEFT_CONTROL) || IsKeyDown(KEY_RIGHT_CONTROL);
+    bool command = IsKeyDown(KEY_LEFT_SUPER);
     bool shift = IsKeyDown(KEY_LEFT_SHIFT) || IsKeyDown(KEY_RIGHT_SHIFT);
+    struct nk_raylib_input_keyboard_check checks[17] = {
```

Comment:
I'm not sure what i was thinking, as this array has 16 elements and below we are only iterating over 12.  I wonder why I did that?

Reply:
Moved it to a define